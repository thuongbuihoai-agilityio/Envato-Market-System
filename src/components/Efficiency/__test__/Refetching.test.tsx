import { render } from '@testing-library/react';

// Components
import EfficiencyRefetch from '../Refetching';

describe('Refetching component test cases', () => {
  it('should render correctly', () => {
    const { container } = render(<EfficiencyRefetch />);

    expect(container).toMatchSnapshot();
  });
});
