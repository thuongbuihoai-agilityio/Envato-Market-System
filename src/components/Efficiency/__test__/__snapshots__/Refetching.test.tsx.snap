// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Refetching component test cases should render correctly 1`] = `
<div>
  <div
    class="css-k0iw6e"
  >
    <div
      aria-valuemax="100"
      aria-valuemin="0"
      class="chakra-progress css-120wkjd"
      data-indeterminate=""
      role="progressbar"
    >
      <svg
        class="css-jf70f3"
        viewBox="0 0 100 100"
      >
        <circle
          class="chakra-progress__track css-1bhxzgy"
          cx="50"
          cy="50"
          r="42"
          stroke-width="10px"
        />
        <circle
          class="chakra-progress__indicator css-6gdwrg"
          cx="50"
          cy="50"
          r="42"
          stroke-width="10px"
        />
      </svg>
    </div>
  </div>
</div>
`;
